Assets/MRTK/Examples/packagetemplate.json

```diff
@@ -28,10 +28,11 @@
     },
     "files": [
         "Samples~*",
-        "StandardAssets*",
-        "package.json.meta",
-        "Microsoft.MixedReality.Toolkit.Examples*",
-        "MRTK.Examples.sentinel*"
+        "StandardAssets~*",
+        "MRTK.Examples*",
+        "License*",
+        "toc*",
```

Comment:
Do we need the toc files? I thought those were only used in our hosted docs

Reply:
they are packaged in the unitypackage files, so i added them.

Reply:
removed